solar_system.rb

```diff
@@ -0,0 +1,137 @@
+# CLASSES
+# SolarSystem Class
+class SolarSystem
+
+  attr_reader :planets
+
+  def initialize(planets, age)
+    @planets = planets
+    @age = age
+  end
+
+  def add_planet(new_planet)
+    @planets << new_planet
+  end
+
+  def planet_list
+    value = ""
+    @planets.each_with_index do |planet, i|
+      value += "\t\t#{i+1}. #{planet.name}\n"
+    end
+    return value
+  end
+
+end
+
+# Planet Class
+class Planet
+
+  attr_reader :name, :diameter, :distance, :year_length, :moons
+
+  def initialize(name, diameter, distance, year_length, moons)
+    @name = name
+    @diameter = diameter
+    @distance = distance
+    @year_length = year_length
+    @moons = moons
+  end
+
+  def attributes
+    return "\t* Planet Name: #{@name}\n\t* Diameter: #{@diameter} km\n\t* Distance From The Sun: #{@distance} km\n\t* Days in a Year: #{@year_length}\n\t* Number of Moons: #{@moons}"
+  end
+
+end
+
+# METHODS
+# Create Method
+
+def create_planet(solarsystem)
+  puts "\nOk. Let's create a planet!"
+
+  #Ask user for input about each attribute.
```

Comment:
I like that you've broken these out as separate methods. I also like that they're _not_ part of your `SolarSystem` class - that would be too much responsibility in one place. Good organization!